src/howitz/endpoints.py

```diff
@@ -129,40 +169,60 @@ def get_event_details(id):
     event_attr = get_event_attributes(int(id))
     event_logs = event_engine.get_log_for_id(int(id))
     event_history = event_engine.get_history_for_id(int(id))
-    # print('EVENT ATTR', event_attr)
-    # print('EVENT LOGS', event_logs)
-    # print('EVENT HISTORY', event_history)
+    app.logger.debug('EVENT ATTRIBUTES %s', event_attr)
+    app.logger.debug('EVENT LOGS %s', event_logs)
+    app.logger.debug('EVENT HISTORY %s', event_history)
```

Comment:
You can compress this to a single line: debug('Event: attrs %s, logs %s, history %s', event_attr, event_logs,event_history). But yeah, I think these can be dropped also. Easy enough to add a breakpoint() later or add them back in.